build.gradle

```diff
@@ -40,7 +40,7 @@ dependencies {
 	implementation fileTree(dir: 'lib', include: '*.jar')
 
 	testShadow group: 'junit', name: 'junit', version: '4.13.2'
-	testShadow group: 'org.easymock', name: 'easymock', version: '5.2.0'
+	testShadow group: 'org.easymock', name: 'easymock', version: '5.0.1'
```

Comment:
it seems like 5.2.0 has a bug. on v5.0.1, the action bar test passes. however, when running this exact same code in v5.2.0, the action bar test encounters the exception described in https://github.com/easymock/easymock/issues/356. that issue was apparently fixed in 5.2.0 but i wonder if it wasn't a complete fix. it could definitely be user error too though. i will probably followup on that issue and see what the maintainers think.

Reply:
https://github.com/easymock/easymock/issues/356#issuecomment-1872626625

Reply:
[#515](https://github.com/easymock/easymock/issues/515)